Docs: Update note on Next.js ignoring CRA files (#67346)
vercel/next.js: docs/02-app/01-building-your-application/11-upgrading/05-from-create-react-app.mdx

}
```

> > **Good to know**: Next.js ignores CRA's `public/manifest.json` file, additional iconography other than the favicon, and [testing configuration](/docs/app/building-your-application/testing), but if these are requirements, Next.js also supports these options. See the [Metadata API](/docs/app/building-your-application/optimizing/metadata) and [Testing](/docs/app/building-your-application/testing) docs for more information.

Author: ```suggestion
> **Good to know**: Next.js ignores CRA's `public/manifest.json` file, additional iconography other than the [favicon](/docs/app/api-reference/file-conventions/metadata/app-icons) and [testing configuration](/docs/app/building-your-application/testing), but if these are requirements, Next.js also supports these options. See the [Metadata API](/docs/app/building-your-application/optimizing/metadata) and [Testing](/docs/app/building-your-application/testing) docs for more information.
```

> > **Good to know**: Next.js ignores CRA's `public/manifest.json` file, additional iconography other than the [favicon](/docs/app/api-reference/file-conventions/metadata/app-icons) and [testing configuration](/docs/app/building-your-application/testing), but if these are requirements, Next.js also supports these options. See the [Metadata API](/docs/app/building-your-application/optimizing/metadata) and [Testing](/docs/app/building-your-application/testing) docs for more information.

Author: ```suggestion
> **Good to know**: Next.js ignores CRA's `public/manifest.json` file, additional iconography (except [favicon](/docs/app/api-reference/file-conventions/metadata/app-icons)), and [testing configuration](/docs/app/building-your-application/testing), but if these are requirements, Next.js also supports these options. See the [Metadata API](/docs/app/building-your-application/optimizing/metadata) and [Testing](/docs/app/building-your-application/testing) docs for more information.
```
